pkgs/tools/system/incron/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchurl, bash }:
+
+stdenv.mkDerivation rec {
+  name = "incron-0.5.12";
+  src = fetchurl {
+    url = "https://github.com/ar-/incron/archive/0.5.12.tar.gz";
+    sha256 = "14cgsfyl43pd86wy40m1xwr7ww023n2jyks66ngybz5s4gbhps6c";
+  };
+
+  patches = [ ./default_path.patch ];
+
+  prePatch = ''
+    sed -i "s|PREFIX = /usr/local|PREFIX = $out|g" Makefile
```

Comment:
Is it not possible to use `makeFlags = [ "PREFIX=$(out)" ];`?

Reply:
Unfortunately this would get overridden:

https://github.com/ar-/incron/blob/master/Makefile#L2

Reply:
Would it? It usually works.

Reply:
I just tested and in this case it does not.

Reply:
Ok, found it: https://www.gnu.org/software/make/manual/html_node/Overriding.html#Overriding `makeFlags` really have greater priority.

Reply:
I'm not sure what the issue is then... I pasted your suggested makeFlags in but get this error:

install -m 0755 -d /usr/local/share/man/man1
install: cannot create directory '/usr': Permission denied

It appears to me in this case the value is not being overridden.

Reply:
Right, the issue is that you are overriding `installPhase`. The [default `installPhase`](https://github.com/NixOS/nixpkgs/blob/b4a5b3efab93e13e31f0ac9b89254f6c2d01e04c/pkgs/stdenv/generic/setup.sh#L1063-L1083) does something like `make $makeFlags $installFlags $installTargets`.

You could switch to something like the following:

```nix
  installFlags = [ "PREFIX=$(out)" ];
  installTargets = [ "install-man" ];

  preInstall = ''
    mkdir -p $out/bin

    # make install doesn't work because setuid and permissions
    # just manually install the binaries instead
    cp incrond incrontab $out/bin/
  '';
```

Reply:
This can be removed now

Reply:
Yeah missed that on the last commit. Thanks for the explanation, that makes sense.